docs/pages/enroll-resources/database-access/database-access.mdx

```diff
@@ -1,49 +1,345 @@
+import mariaDbSvg from '@site/src/components/Icon/svg/maria-db.svg';
+import redshiftSvg from '@site/src/components/Icon/svg/amazon-redshift.svg';
+import mongoDbSvg from '@site/src/components/Icon/svg/mongodb.svg';
 
-Some of the things you can do with database access:
+<DocHero
+  title="Enroll your first database"
+  youtubeVideoId="rPX_973PoW8"
+  ctaLinks={[
+    {
+      title: "Get started with database access",
+      href: "./getting-started/",
+      variant: "secondary",
+      arrow: true,
+    },
+  ]}
+  links={[
+    {
+      title: "Access control",
+      description: "Apply RBAC to databases for users to access required data.",
+      href: "./rbac/",
+      icon: userCheckSvg,
+    },
+    {
+      title: "Troubleshooting",
+      description: "Common Database access issues with Teleport and their solutions.",
+      href: "./troubleshooting/",
+      icon: questionSvg,
+    },
+  ]}
+>
+The Teleport Database Service provides secure database access with short-lived certificates via SSO, RBAC with Access Requests, and audit logs. Improve access control and visibility while centralizing management across infrastructure.
+</DocHero>
 
-- Enable users to retrieve short-lived database certificates using a Single Sign-On
-  flow, thus maintaining their organization-wide identity.
-- Configure role-based access controls for databases and implement custom
-  [Access Request](../../identity-governance/access-requests/access-requests.mdx) workflows.
-- Capture database activity in the Teleport audit log.
+<UseCasesList
+  title="Database guides"
+  desktopColumnsCount={1}
+  itemLayout="row"
+  useCases={[
+    {
+      title: "AWS databases",
+      description: "Enroll and protect AWS managed databases.",
+      href: "./enroll-aws-databases/",
+      tags: [
+        {
+          name: "DocumentDB",
+          href: "./enroll-aws-databases/aws-docdb/",
+          icon: "aws",
+        },
+        {
+          name: "DynamoDB",
+          href: "./enroll-aws-databases/aws-dynamodb/",
+          icon: "amazonDynamoDb",
+        },
+        {
+          name: "ElastiCache for Redis and Valkey",
+          href: "./enroll-aws-databases/redis-aws/",
+          icon: "amazonElastiCache",
+        },
+        {
+          name: "MemoryDB",
+          href: "./enroll-aws-databases/aws-memorydb/",
+          icon: "amazonMemoryDb",
+        },
+        {
+          name: "ElastiCache Serverless for Redis and Valkey",
+          href: "./enroll-aws-databases/elasticache-serverless/",
+          hidden: true,
+          icon: "amazonElastiCache",
+        },
+        {
+          name: "Keyspaces (Apache Cassandra)",
+          href: "./enroll-aws-databases/aws-cassandra-keyspaces/",
+          hidden: true,
+          icon: "amazonKeyspaces",
+        },
+        {
+          name: "OpenSearch",
+          href: "./enroll-aws-databases/aws-opensearch/",
+          hidden: true,
+          icon: "amazonOpenSearch",
```

Comment:
same question for opensearch, could we use the icon from the same set?
https://aws-icons.com/icons/opensearch-service